<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PDF Eksport</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ExportDialog {
  background: black;
}

#ExportDialog QLineEdit {
  color: gray;
  border: 2px solid lightgray;
  border-radius: 5px;
}

#ExportDialog QLineEdit:focus {
  color: gray;
  border: 3px solid rgb(8,142,140);
  border-radius: 5px;
}

#ExportDialog {
  background: black;
}

#ExportDialog QLineEdit {
  color: gray;
  border: 2px solid lightgray;
  border-radius: 5px;
}

#ExportDialog QLineEdit:focus {
  color: gray;
  border: 3px solid rgb(8,142,140);
  border-radius: 5px;
}

QCalendarWidget { 
font-size:10pt; 
font-family:Calibri;
}







</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>233</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: gray;
border: 2px solid rgb(20,20,20);
border-radius: 5px;
background: rgb(35,35,35);
}

QPushButton:hover {
color: white;
border: 2px solid rgb(20,20,20);
border-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
   stop: 0 rgba(80,80,80), stop: 0.7 rgb(35,35,35));
}

QPushButton:pressed {
color: white;
border: 2px solid rgb(20,20,20);
border-radius: 5px;
background: rgb(80,80,80);
}</string>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>233</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: gray;
border: 2px solid rgb(20,20,20);
border-radius: 5px;
background: rgb(35,35,35);
}

QPushButton:hover {
color: white;
border: 2px solid rgb(20,20,20);
border-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
   stop: 0 rgba(80,80,80), stop: 0.7 rgb(35,35,35));
}

QPushButton:pressed {
color: white;
border: 2px solid rgb(20,20,20);
border-radius: 5px;
background: rgb(80,80,80);
}</string>
   </property>
   <property name="text">
    <string>Eksportuj</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pathLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>415</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Plik docelowy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pathButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>55</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: gray;
border: 2px solid rgb(20,20,20);
border-radius: 5px;
background: rgb(35,35,35);
}

QPushButton:hover {
color: white;
border: 2px solid rgb(20,20,20);
border-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
   stop: 0 rgba(80,80,80), stop: 0.7 rgb(35,35,35));
}

QPushButton:pressed {
color: white;
border: 2px solid rgb(20,20,20);
border-radius: 5px;
background: rgb(80,80,80);
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startDateButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>55</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border:none;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/calendar.png</normaloff>:/images/images/calendar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="stopDateButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>55</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border:none;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/calendar.png</normaloff>:/images/images/calendar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="startDateLineEdit">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>80</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Data początkowa</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stopDateLineEdit">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>80</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Data końcowa</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
